src/odin_devices/si5338.py

```diff
@@ -723,17 +723,17 @@ def _write_register_map(self, filepath, verify=False):
                 for line in lines:
                     # if the line is empty, skip it
                     if line == "":
-                        continue
+                        continue  # pragma: no cover
                     # if the line starts with a #, it is a comment and should be skipped
                     if line.strip()[0] == "#":
-                        continue
+                        continue  # pragma: no cover
                     # if the line does not provide an address and value, skip it
                     if len(line.split(",")) != 2:
-                        continue
+                        continue  # pragma: no cover
                     address = int(line.split(",")[0])
                     # Skip address 27 again because it is skipped in the writing stage
                     if address == 27:
-                        continue
+                        continue  # pragma: no cover
```

Comment:
As above, these lines aren't covered, and the coverage should reflect this unless there is a specific reason why the code cannot be run in test.